iis-common/src/test/java/eu/dnetlib/iis/common/AbstractOozieWorkflowTestCase.java

```diff
@@ -27,7 +27,6 @@
  * @author madryk
  *
  */
-@IntegrationTest
```

Comment:
Yes, you're right, we could drop `IntegrationTest` from each workflow test and at the same time leave the annotation on `AbstractOozieWorkflowTestCase`. It would simplify workflow test classes. I think it's an individual matter, I'd prefer to place the annotation on the child class and thats why I chose this solution. But super class is also fine. One point for annotating super class is that while working on this task I found two workflow test classes in `iis-wf-transformers`extending `AbstractOozieWorkflowTestCase` and not marked as `IntegrationTest`. They where not excluded from running because previously `AbstractOozieWorkflowTestCase` was marked with integration test category. Summerizing, it's a good idea and I'm applying it :)

Reply:
I'm also adding `Inherited` annotation on `IntegrationTest` to indicate that `AbstractOozieWorkflowTestCase` annotation will be inherited by extending classes.